ersilia/hub/content/card.py

```diff
@@ -704,30 +706,33 @@ def get(self, model_id):
 class LocalCard(ErsiliaBase):
     def __init__(self, config_json):
         ErsiliaBase.__init__(self, config_json=config_json)
-
+    
+    @lru_cache(maxsize=32)
     def get(self, model_id):
-        model_path = self._model_path(model_id)
-        card_path = os.path.join(model_path, CARD_FILE)
+        model_path = os.path.join(EOS,  "dest", model_id)
+        card_path = os.path.join(model_path, INFORMATION_FILE)
         if os.path.exists(card_path):
             with open(card_path, "r") as f:
-                card = json.load(f)
+                data = json.load(f)
+            card = data.get("card")
             return card
         else:
             return None
-            
+        
+    @lru_cache(maxsize=32)        
     def get_service_class(self, model_id):
         """
         This method returns information about how the model was fetched by reading 
         the service class file located in the model's bundle directory. If the service 
         class file does not exist, it returns None.
         """
-        service_class_path = os.path.join(
-                self._get_bundle_location(model_id), SERVICE_CLASS_FILE
-            )
+        model_path = os.path.join(EOS,  "dest", model_id)
+        service_class_path = os.path.join(model_path, INFORMATION_FILE)
```

Comment:
I think it's better to call it 'information_file_path' because calling it 'service_class_path' is misleading since it is not the path to the SERVICE_CLASS_FILE, but to the INFORMTION_FILE.